### PR TITLE
Create pop-up window

### PR DESCRIPTION
**Pull request by RaphDasilva**
-My partner and I were able to do the following:

--We updated the HTML file to remove the static content in the project section.
--We updated the JS file to add the projects to the page dynamically.
--We updated the JS file to add a function pop-up and display its content.
--We made use of Linters to fix the CSS and JavaScript errors.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @RaphDasilva,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- You have submitted a project, that has no software license. Kindly add one and make sure the link on the Readme file works fine. Also please add .vscode to your .gitignore file to ignore it. This will keep your repository organized and professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -63,80 +70,69 @@ <h1 class="header-text">Hey there. I’m Raphael
...
+                                when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+                        </div>
+                        <div class="pop-buttons">
+                            <button type="submit" class="see-live">See Live <img src="img/seeliveicon.svg"
+                                    alt="see live icon"></button>
+                            <button type="submit" class="see-source">See Source <img src="img/Vector.png"
+                                    alt="see Source icon"></button>
+                        </div>
+                    </div>
+                </div>
```
Comment (amejid) [inline index.html]: - [ ] Good job generating the projects section dynamically using javascript. But the popup is hardcoded, I am seeing the same popup for all projects. Kindly utilize the data stored in the array of objects to generate your popup dynamically. The data in the array of objects should be reflected in the popup. ✍️
Check the example below to structure your array of objects.

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

File: script.js [hunk 2]
```diff
@@ -6,4 +6,75 @@ openMenu();
...
+
+closePopUp();
+
+const recentWorks = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    skillsDisplay: ['Ruby on rails', 'CSS', 'JavScript', 'html'],
+    submitButton: 'See Project',
+    upCardimg: '#!',
+  },
```
Comment (amejid) [inline script.js]: - [ ] Kindly add all the necessary information required to generate your popup such as `image URL, description, live link`. 🧐️
You should generate the popup dynamically using the data stored here.

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @RaphDasilva, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Top-notch UI design :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @RaphDasilva 👋, 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**